lib/jsdom/living/postmessage.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+const isValidTargetOrigin = require("../utils").isValidTargetOrigin;
+
+module.exports = function (message, targetOrigin) {
+  let window = this;
+  let event;
+
+  if (!targetOrigin) {
+    throw new TypeError("'postMessage' requires a second argument 'targetOrigin'");
+  }
+
+  if (!isValidTargetOrigin(targetOrigin)) {
+    throw new SyntaxError("Failed to execute 'postMessage' on 'Window': " +
+      "Invalid target origin '" + targetOrigin + "' in a call to 'postMessage'.");
+  }
+
+  if (targetOrigin !== "*" && targetOrigin !== window.origin) {
+    return;
+  }
+
+  event = window.document.createEvent("MessageEvent");
```

Comment:
Actually it looks like you need to implement a new Event interface: https://html.spec.whatwg.org/multipage/comms.html#messageevent can be its own file in living.